interface/docker_build_interface.sh

```diff
@@ -0,0 +1,22 @@
+docker build -t adage-server/docker-interface-base -f Dockerfile.base .
```

Comment:
What happens if `adage-server/docker-interface-base` is already up to date? Does this step needlessly re-build it?